src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -580,9 +580,9 @@ def get_assisted_controller_status(kubeconfig):
 
 
 def download_iso(image_url, image_path):
```

Comment:
I'm not sure we need that method.
On the REST flow we are using [download_file](https://github.com/openshift/assisted-test-infra/blob/3df9de413412d2436b313a696827804ae892f756/src/assisted_test_infra/test_infra/utils/utils.py#L38) method.
Maybe we want to consider to try to use it here instead of duplicate the logic

Reply:
Fair point, I will look into that.